.circleci/config.yml

```diff
@@ -273,18 +276,43 @@ jobs:
   build_devtools_and_process_artifacts:
     docker: *docker
     environment: *environment
-    parallelism: 20
     steps:
       - checkout
       - attach_workspace: *attach_workspace
+      - *restore_yarn_cache
       - *restore_node_modules
+      - run:
+          name: Install Packages
+          command: yarn --frozen-lockfile --cache-folder ~/.cache/yarn
       - run:
           environment:
             RELEASE_CHANNEL: experimental
           command: ./scripts/circleci/pack_and_store_devtools_artifacts.sh
       - store_artifacts:
           path: ./build/devtools.tgz
 
+  build_devtools_scheduling_profiler:
+    docker: *docker
+    environment: *environment
+    steps:
+      - checkout
+      - attach_workspace: *attach_workspace
+      - *restore_yarn_cache
+      - *restore_node_modules
+      - run:
```

Comment:
Shouldn't this be tagged with `RELEASE_CHANNEL: experimental`?

The scheduling profiler uses `unstable_createRoot`. The stable `yarn_build` command doesn't build a version of ReactDOM that exposes this API.

Reply:
I don't think so, unless I'm missing something. The `yarn_build` CI job builds everything with the experimental channel. Because we depend on those packages, I don't think the scheduling profiler's build will be affected by a `RELEASE_CHANNEL` tag.

I've also confirmed that the built artifact works:

![image](https://user-images.githubusercontent.com/12784593/93787327-ecea6400-fc62-11ea-87ef-c368d9c26d96.png)


![image](https://user-images.githubusercontent.com/12784593/93787080-9f6df700-fc62-11ea-9c1a-1cc18fafed41.png)

Reply:
Oh, huh. I guess I'm a little out of date on our how Circle CI config is setup. `build_devtools_and_process_artifacts` specifies `RELEASE_CHANNEL: experimental`, but maybe it's unnecessary since, as you say, `yarn_build` is in the experimental channel (and `RELEASE_CHANNEL_stable_yarn_build` is stable)

Reply:
Yeah, there's a good chance that that's unnecessary, but I didn't investigate that as it was already there 😆